run.py

```diff
@@ -44,10 +44,12 @@ def install_secret_keys(application, secret_session='secret_key', secret_csrf='s
     If the file does not exist, print instructions to create it from a shell with a random key, then exit.
     """
     do_exit = False
-    session_file = os.path.join(application.root_path, secret_session)
-    csrf_file = os.path.join(application.root_path, secret_csrf)
+    session_file_path = os.path.join(application.root_path, secret_session)
+    csrf_file_path = os.path.join(application.root_path, secret_csrf)
     try:
-        application.config['SECRET_KEY'] = open(session_file, 'rb').read()
+        session_file = open(session_file_path, 'rb')
```

Comment:
I'd rather see a `with open... as x`, as that'll automatically take care of closing the file once it goes out of context.

Reply:
Done! Thanks for the tip! :)